Lesson08_Task01.py

```diff
@@ -0,0 +1,36 @@
+"""
+Lesson08 Task01.
+
+Given the list of integers ( positive , negative, zeros)
+
+Find max multiplication of three values in list
+l1 = [-7, 1, 2, 0, 3, 12, -24, 5, -1, 6]
+
+Input: [-1, 1, 2, 0, 3, 12, 4, 5, -1, 6]
+Output: Max value = "360". Nums are: (12, 5, 6)
+
+Input: [-7, 1, 2, 0, 3, 12, -24, 5, -1, 6]
+Output: Max value = "2016". Nums are: (-7, 12, -24)
+"""
+
+l1 = [-7, 1, 2, 0, 3, 12, -24, 5, -1, 6]
+
+copy_l1 = l1[:]
+
+# check if the length of the list contains at least 3 val
+if len(copy_l1) < 3:
```

Comment:
Such blocks better orginize as single if with valueError inside. This is something new - but just for your information

```
if len(copy_l1) < 3:
    raise ValueError('At least 3 values should be in the input') 
```

And no else block required after this ^^

Reply:
created new file with this solution